app/Http/Middleware/Authenticate.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Middleware;
+use Illuminate\Auth\Middleware\Authenticate as Middleware;
+
+class Authenticate extends Middleware
+{
+    /**
+     * Get the path the user should be redirected to when they are not authenticated.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return string
+     */
+    protected function redirectTo($request)
+    {
+        if (! $request->expectsJson()) {
+            return route('home');
+        }
```

Comment:
There are some instances where if you go through the OAuth process starting signed out, after you sign in, it takes you to the home page.  I'm fairly certain this middleware is causing that but I need to figure out how to make it redirect

But otherwise it not returning anything didn't seem to cause errors

Reply:
To clarify and expand what I sent in Slack, this is actually just overriding a function that's supposed to return a string.  It's part of an exception handler behind the scenes that's deciding where to send a user if they are logged out, basically